### ISSUE TITLE
Realtime Price Display

### ISSUE THREAD

**Issue by RogerHuangPKX** [CSCI-GA-2820-SU24-001/shopcarts#6]
**As a** shopping cart user,
**I need** to get the newest price of items in shopping cart,
**So that** I could know which item gets more discount
   
 ### Details and Assumptions
 * •	Determine customer id, cart id, and product id of the product to get latest price
   
 ### Acceptance Criteria  
   
 ```gherkin
 Given database and service are running
 When the shopping cart is opened
 Then the shopping cart should look up all the items with the latest price and discounts.
```

**Comment by manisha-goyal** [comment]
Based on professor's feedback, can we remove this story? Since we will not be able to integrate with the products service, it feels like it will be unnecessary effort. Rather, we can focus on doing the required stories well first.

**Comment by AirKristo** [comment]
Yes I agree, if easier we can put this as unused or reuse so we can just edit the story later for something else. Not sure if it will let us delete.

**Comment by ZhongKai-07** [comment]
Agree too
